gnucash/import-export/import-main-matcher.h

```diff
@@ -36,6 +36,8 @@
 #include "Transaction.h"
 #include "import-backend.h"
 
+#define IMPORT_MAIN_MATCHER_CM_CLASS "transaction-matcher-dialog"
```

Comment:
Does this really need to be in the header?

Reply:
No, I only replicated what I saw was being done elsewhere....

Reply:
Doing a search for _CM_CLASS return about 250 entries with the vast majority ending in .c files so I would move this.